src/dos/program_imgmount.cpp

```diff
@@ -387,10 +387,10 @@ void IMGMOUNT::Run(void)
 				WriteOut(MSG_Get("PROGRAM_IMGMOUNT_INVALID_IMAGE"));
 				return;
 			}
-			fseek(diskfile, 0L, SEEK_END);
-			uint32_t fcsize = (uint32_t)(ftell(diskfile) / 512L);
+			cross_fseeko(diskfile, 0L, SEEK_END);
+			uint32_t fcsize = static_cast<uint32_t>(cross_ftello(diskfile) / 512L);
 			uint8_t buf[512];
-			fseek(diskfile, 0L, SEEK_SET);
+			cross_fseeko(diskfile, 0L, SEEK_SET);
```

Comment:
If `cross_fseeko` fails, `diskfile` is in bad shape. 
Suggest checking `cross_fseeko`'s result and bailing out (similar to the subsequent `fread`).